.evergreen/atlas_data_lake/drivers-test-local.yaml

```diff
@@ -0,0 +1,66 @@
+environment: development
+
+hosting:
+  frontend:
+    bindAddrs: ["localhost:27017"]
+    cursor:
+      metadata:
+        memory: true
+  showInternalErrors: true
+
+execution:
+  mqlrunPath: ${MONGOHOUSE_MQLRUN}
+
+tenants:
+  config:
+    inline:
+      hostnames:
+        - localhost
+      security:
+        roles:
+          # Based on the atlasAdmin custom role defined in NDSDBUser.java in the
+          # 10gen/mms repo.
+          - role: atlasAdmin
+            roles:
+              - role: readWriteAnyDatabase
+              - role: enableSharding
+              - role: dbAdminAnyDatabase
+              - role: backup
+              - role: clusterMonitor
+            privileges:
+              - actions: ["viewRole", "moveChunk", "splitChunk", "viewUser"]
+              - actions: ["flushRouterConfig", "storageGetConfig", "storageSetConfig"]
+                resource:
+                  cluster: true
+        users:
+          - name: mhuser # password: "pencil"
+            credentials:
+              scramSHA1:
+                iterations: 10000
+                salt: "CVmjg1IDp3KfHfiXNZhmbQ=="
+                storedKey: "9cIonYrtLI/+OmZ96wiQkduwXeI="
+                serverKey: "kCfqkc7sgQaihzXHqbxZxz3O3H4="
+              scramSHA256:
+                iterations: 15000
+                salt: "6i+wBCHE21EaI8IF9+tvPkCoYWY+IyedLaVXGQ=="
+                storedKey: "hnn0iJZ6ZNO9X34oyxtZUKLAMScwpCdch3vGPuDXOvM="
+                serverKey: "3Qg6ptTiTvPwhGSZi0NpAkmsW8P5C/J46Is1t7xaIho="
+            roles:
+              - role: atlasAdmin
+      storage:
+        stores:
+          - name: localstore
+            provider: local
+            path: "${DRIVERS_TOOLS}/.evergreen/atlas_data_lake/testdata/stores/drivers-tests"
+        databases:
+          - name: test
+            collections:
+              - name: driverdata
+                dataSources:
+                  - storeName: localstore
+                    path: /test/*
+  sqlSchema:
+    inline:
+      - database: "test"
+        collection: "driverdata"
+        schemaFile: "${DRIVERS_TOOLS}/.evergreen/atlas_data_lake/schema/testdata/driver-data.json"
```

Comment:
In the interest of simplicity, can we refer to `.evergreen/atlas_data_lake/schema.json` or `.evergreen/atlas_data_lake/schema/test.driverdata.json` instead? I don't think "testdata" appears anywhere else, so I assume that doesn't provide much context. Also, I think it'd be clearer to use the namespace in the schema filename.

Likewise, can `drivers-test-local.yaml` be renamed to `config.yaml`. That would be self-documenting by immediately looking at the directory, and I don't imagine we'd need multiple configuration files for this project.

Reply:
The `sqlSchema` section can be removed completely if we are _not_ using the `$sql` stage in an aggregation pipeline.

Reply:
None of the `aggregation` tests in https://github.com/mongodb/specifications/pull/821 use `$sql`, so I believe this can probably be removed entirely.